### ISSUE TITLE
Can't search for gene families associated with a gene model

### ISSUE THREAD

**Issue by ekcannon** [ncgr/tripal_phylogeny#21]
Yes, this can be done via the gene search page, but if a user is focused on gene families, it may not be obvious to go use the gene search page instead.

**Comment by adf-ncgr** [comment]
yes, this a good one, but also a bit subtle due to the somewhat heterogeneous nature of the way we handle gene families. Ideally, such a search would be able to find families for:
* genes that have been assigned to families but are not included in the trees
* genes from outgroup species that are included in the trees, but not accessible via the gene search page (and currently, we don't actually store "genes" for these, just the "polypeptides" that are in the trees)- supporting search for families by arabidopsis genes, for example, would be very nice

if we stick with the current drupal-views-based approach, then we'll probably need to put the genelist for a family into a tsvector in the row for the gene family. in any case, since I think we won't be able to rely on things in the family being in the tree, however we accomplish this should probably access the association through the featureprop rows that represent gene family assignments.
